### ISSUE TITLE
Add "Next Review" / "Prev Review" keyboard actions on "Review List" view

### ISSUE THREAD

**Issue by al-the-x** [Reviewable/Reviewable#461]
I love all the keyboard shortcuts that exist on the "Review Detail" view. Could we have some bindable actions on the "Review List" view, though? Right now when I land on `/reviews` the focus is given to the "Search" input, so I can filter down my reviews, but then I have to `TAB` through the links in the list to get to the review that I want. I find myself viewing the list, typing enough to limit the choices to just one, then `TAB`-ing through to the detail link.

Instead, if there was an bindable action that allowed me to highlight the next or previous Review in the list and to `focus` / `blur` the "Search" input as desired, I could update my keybindings to make it suit my needs (to be more like `vim`, as in all things). GitHub, Gmail, and Google Inbox do something similar whereby I can press a button to change the highlighted row, then press `Enter` to select that item's detail view.

**Comment by pkaminski** [comment]
Agreed, something like this would be much better.  Unfortunately, the current implementation is limited to custom keybindings on the review page only, and the widget set I'm using isn't very keyboard-friendly.  I'll look into this as part of the move from Angular to Vue.
